plain_http_producer.go

```diff
@@ -10,44 +9,47 @@ import (
 	"strings"
 	"time"
 
-	"github.com/Financial-Times/go-logger"
+	log "github.com/Financial-Times/go-logger/v2"
 	queueProducer "github.com/Financial-Times/message-queue-go-producer/producer"
 )
 
 type plainHTTPMessageProducer struct {
 	config queueProducer.MessageProducerConfig
 	client plainHttpClient
+	logger *log.UPPLogger
 }
 
 type plainHttpClient interface {
 	Do(req *http.Request) (resp *http.Response, err error)
 }
 
 // newPlainHTTPMessageProducer returns a plain-http-producer which behaves as a producer for kafka (writes messages to kafka), but it's actually making a simple http call to an endpoint
-func newPlainHTTPMessageProducer(config queueProducer.MessageProducerConfig) queueProducer.MessageProducer {
-	cmsNotifier := &plainHTTPMessageProducer{config, &http.Client{
-		Timeout: 60 * time.Second,
-		Transport: &http.Transport{
-			MaxIdleConnsPerHost: 100,
-			Dial: (&net.Dialer{
-				KeepAlive: 30 * time.Second,
-			}).Dial,
-		}}}
+func newPlainHTTPMessageProducer(config queueProducer.MessageProducerConfig, logger *log.UPPLogger) queueProducer.MessageProducer {
+	cmsNotifier := &plainHTTPMessageProducer{
+		config: config,
+		client: &http.Client{
+			Timeout: 60 * time.Second,
+			Transport: &http.Transport{
+				MaxIdleConnsPerHost: 100,
+				Dial: (&net.Dialer{
+					KeepAlive: 30 * time.Second,
+				}).Dial,
+			}},
+		logger: logger}
 	return cmsNotifier
 }
 
-func (c *plainHTTPMessageProducer) SendMessage(uuid string, message queueProducer.Message) (err error) {
-	req, err := http.NewRequest("POST", c.config.Addr+"/notify", strings.NewReader(message.Body))
+func (p *plainHTTPMessageProducer) SendMessage(uuid string, message queueProducer.Message) (err error) {
```

Comment:
Can we remove the `err` name from the return?
```suggestion
func (p *plainHTTPMessageProducer) SendMessage(uuid string, message queueProducer.Message) error {
```